packages/typescript/src/internal/build_defs.bzl

```diff
@@ -22,7 +22,8 @@ load("@build_bazel_rules_typescript//internal:common/compilation.bzl", "COMMON_A
 load("@build_bazel_rules_typescript//internal:common/tsconfig.bzl", "create_tsconfig")
 load("//internal:ts_config.bzl", "TsConfigInfo")
 
-_DEFAULT_COMPILER = "@npm//@bazel/typescript/bin:tsc_wrapped"
+# NB: substituted with "@npm//@bazel/typescript/bin:tsc_wrapped" in the pkg_npm rule
+_DEFAULT_COMPILER = "@build_bazel_rules_typescript//internal:tsc_wrapped_bin"
```

Comment:
Does `@build_bazel_rules_typescript` work different then the other `@npm_bazel_*` targets? So we can't stick with the `@npm//@bazel/X` => `@npm_bazel_X` convention here?

Reply:
yeah it's a weird snowflake because rules_typescript has a g3 source-of-truth